packages/legacy-scripting-runner/test/integration-test.js

```diff
@@ -3361,6 +3419,43 @@ describe('Integration Test', () => {
       });
     });
 
+    it('file upload, KEY_pre_write, cancel multipart', () => {
+      const appDefWithAuth = withAuth(appDefinition, apiKeyAuth);
+      appDefWithAuth.legacy.scriptingSource =
+        appDefWithAuth.legacy.scriptingSource.replace(
+          'file_pre_write_cancel_multipart',
+          'file_pre_write'
+        );
+      const compiledApp = schemaTools.prepareApp(appDefWithAuth);
+      const app = createApp(appDefWithAuth);
+
+      const input = createTestInput(
+        compiledApp,
+        'creates.file.operation.perform'
+      );
+      input.bundle.authData = { api_key: 'secret' };
+      input.bundle.inputData = {
+        filename: 'pig.png',
+        file: `${HTTPBIN_URL}/image/png`,
+        yes: true,
+      };
+      return app(input).then((output) => {
+        const response = output.results;
+
+        const contentType = response.headers['Content-Type'][0];
+        contentType.should.startWith('application/json');
+
+        const json = response.json;
+        should.deepEqual(json.file, [
+          'png',
+          `${HTTPBIN_URL}/image/png`,
+          'image/png',
+        ]);
+        should.equal(json.filename, 'pig.png');
+        should.equal(json.yes, true);
+      });
+    });
```

Comment:
This is the only meaningful change in `integration-test.js`. Others are only auto-formatting done by prettier.